modules/javafx.web/src/main/native/Source/ThirdParty/libxml/src/dict.c

```diff
@@ -904,30 +924,91 @@ xmlDictQLookup(xmlDictPtr dict, const xmlChar *prefix, const xmlChar *name) {
  * Pseudo-random generator
  */
 
+#ifdef _WIN32
+  #define WIN32_LEAN_AND_MEAN
+  #include <windows.h>
+  #include <bcrypt.h>
+#else
+  // This block will compile on macOS (and any non-Linux system) if HAVE_GETENTROPY is defined
+  #if defined(HAVE_GETENTROPY) && !defined(__linux__)
```

Comment:
Can you check whether this is needed if `HAVE_GETENTROPY` is left undefined? I suspect it will no longer be necessary, and it would be better not to have local mods to upstream files.

If a modification _is_ needed, then we will need a clear comment with the changes, noting that this is a JavaFX-specific addition.

Reply:
removed custom change